lib/js/patterns/tabs.js

```diff
@@ -84,6 +84,9 @@ function tabs () {
 
   function getOptions (e) {
     var tab = e.target;
+    if(!classy.has(tab, 'tab-title')){
```

Comment:
I think you could just use `tab = e.currentTarget` instead of performing this lookup?

Reply:
yup that worked too